packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -321,21 +467,139 @@ export default function ViewApplicationPage() {
+                round?.payoutStrategy.strategyName != ROUND_PAYOUT_DIRECT && (
+                  <ApplicationOpenDateRange round={round} />
+                )}
+              {round && <RoundOpenDateRange round={round} />}
+              <div className="absolute right-0">
+                <ViewGrantsExplorerButton
+                  iconStyle="h-4 w-4"
+                  chainId={`${chain.id}`}
+                  roundId={id}
+                />
+              </div>
+            </div>
+          </header>
+          <div className="container mx-auto h-screen px-4 pb-7">
+            <main className="flex flex-row">
+              {/* Sidebar */}
+              {(application?.statusSnapshots || []).length > 0 && (
+                <div
+                  className="w-24 basis-1/6 border-r pt-12"
+                  data-testid="sidebar-steps-container"
+                >
+                  <div className="flex flex-col">
+                    {application
+                      .statusSnapshots!.sort((a, b) =>
+                        moment(a.timestamp).diff(moment(b.timestamp))
+                      )
+                      .map((s, index) => (
+                        <Step
+                          key={index}
+                          status={getStepStatus(
+                            s.status,
+                            index === application.statusSnapshots!.length - 1
+                          )}
+                          title={s.statusDescription.toLowerCase()}
+                          icon={
+                            <CalendarIcon className="text-grey-400 h-3 w-3" />
+                          }
+                          text={
+                            <>
+                              <div className="mb-[2px]">
+                                {moment(s.timestamp).format("MMMM Do YYYY")}
+                              </div>
+                              <div>{getUTCTime(s.timestamp)}</div>
+                            </>
+                          }
+                          index={index}
+                        />
+                      ))}
+                    {/* When is direct round and application is in review */}
+                    {application?.payoutStrategy?.strategyName ==
+                      ROUND_PAYOUT_DIRECT &&
+                      application.status === "PENDING" &&
+                      !application.inReview && (
+                        <>
+                          <Step
+                            title="In review"
+                            icon={
+                              <CalendarIcon className="text-grey-400 h-3 w-3" />
+                            }
+                            text="Waiting"
+                          />
+                        </>
+                      )}
+                    {/* When application is not yet evaluated */}
+                    {!["APPROVED", "REJECTED"].includes(
+                      application.status as string
```

Comment:
If `status` is a string, string union, or a string enum, it doesn't need to be cast as `string`. If it's something else, it can't be either `"APPROVED"` or `"REJECTED"`.